site/docs/samples/fog-protect.md

```diff
@@ -0,0 +1,68 @@
+# Dashboard sample
+
+This use-case demonstrates how Fybrik uses runtime evaluation of policies 
+to control access privileges for multiple assets.
+
+## Mockup version of the real scenario
+
+This is based on the smart manufacturing scenario of *Fogprotect*, 
+where there exists a manufacturing factory which can be controlled/supervised via 
+transmitted commands, without the need of the staff member to be physically present 
+in the factory.  
+The scenario here is simplified for the sake of the demo. The resources in this demo are:  
+1. A robot in the manufacturing area that can be started/stopped.  
+2. Two factory sectors, a production and a non-production sector, where some employees wear helmets 
+and some don't. There's a video camera with image processing which monitors and counts the number of employees 
+wearing/not wearing helmets in each sector. This is called the safety data of the factory.  
+
+There are multiple roles in this scenario:  
+1. A *Foreman* who is allowed to access all of the assets because he is the foreman of the factory. 
+Furthermore, only the Foreman is allowed to control the robot in the manufacturing area.  
+2. A *Worker* who is not allowed to control the robot, and doesn't have privileges to see the number 
+of employees wearing/not wearing helmets in each of the available sectors. However, a Worker can see the total 
+number of employees in each sector.  
+3. An *HR* who is also not allowed to control the robot, but has access to view the number of employees 
+wearing/not wearing helmets in each sector.
+
+## Architecture
+The project contains 3 main components:  
+- A backend data server, responsible for reading/writing data (possibly to a database).  
+- A proxy server, responsible for intercepting HTTP requests sent
```

Comment:
is this the fybrik module? if so, please describe it as a module and provide a reference to the github where the module reside.